### PR TITLE
Code docs updates

### PR DESCRIPTION
**Pull request by DariuszDepta**

### CODE REVIEW

File: src/addons/addresses/mock.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ use cosmwasm_std::{instantiate2_address, Addr, Api, CanonicalAddr, Storage};
 use sha2::digest::Update;
 use sha2::{Digest, Sha256};
 
-/// Address generator that mimics the original `wasmd` behavior.
+/// Address generator that mimics the [wasmd](https://github.com/CosmWasm/wasmd) behaviour.
```
Comment (jawoznia) [inline src/addons/addresses/mock.rs:7]: nit: `behavior` was a proper form

File: src/app.rs [hunk 2]
```diff
@@ -362,7 +361,7 @@ where
         self.block = block;
     }
 
-    // this let's use use "next block" steps that add eg. one height and 5 seconds
+    /// Advances the current block to "next block" steps and adds one height and 5 seconds.
```
Comment (jawoznia) [inline src/app.rs:364]: I think this comment is incorrect. This method updates the block info by applying some closure.

This comment should refer to the `next_block` function.

File: src/app.rs [hunk 3]
```diff
@@ -482,6 +486,7 @@ where
     GovT: Gov,
     StargateT: Stargate,
 {
+    /// Returns a querier populated with the instance if this [Router].
```
Comment (jawoznia) [inline src/app.rs:489]:
```suggestion
    /// Returns a querier populated with the instance of this [Router].
```

**Review by jawoznia — APPROVED** [review]

File: src/addons/addresses/mock.rs (hunk 1 above)
Comment (DariuszDepta) [reply src/addons/addresses/mock.rs]: Changed to US form: behavior.

File: src/app.rs (hunk 2 above)
Comment (DariuszDepta) [reply src/app.rs]: Right, fixed.

File: src/app.rs (hunk 3 above)
Comment (DariuszDepta) [reply src/app.rs]: Yeah. Fixed.

**Pull request merged**